src/utils/isEqual.ts

```diff
@@ -1,4 +1,4 @@
-export function isEqual<T>(identify: (t: T) => string, a: T, b: T) {
```

Comment:
To resolve `TS9007: Function must have an explicit return type annotation with --isolatedDeclarations`, add return type explicity.